electrophysiology/models.py

```diff
@@ -106,21 +118,26 @@ class SpikeSortedUnit(BaseExperimentalData):
     channel_group = models.IntegerField(null=True, blank=True,
                                         help_text="which shank this unit came from (an integer not a link)")
     trough_to_peak_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, computed from unfiltered mean waveform. TODO: which waveform?")
+                                 help_text="ms, computed from unfiltered mean spike waveform.")
     half_width = models.FloatField(null=True, blank=True,
-                                 help_text="ms, half width of negative peak in unfiltered waveform. TODO: which waveform?")
+                                 help_text="ms, half width of negative peak in unfiltered spike waveform.")
     trough_to_peak_amplitude = models.FloatField(null=True, blank=True,
-                                 help_text="µV, from filtered waveform. TODO: which waveform?")
+                                 help_text="µV, from filtered spike waveform.")
+								 # when you say which waveform, do you mean which channel? I guess the peak, but we can leave it unspecified...
```

Comment:
I don't think we should specify that level of detail at this stage. 

This sort of thing will vary between spike sorting software. A description of what was done can go in the "provenance directory" and custom json fields.